tests/withIdleTimer.test.tsx

```diff
@@ -1,4 +1,4 @@
-import { Component, ReactNode } from 'react'
+import { Component, createRef, ReactNode } from 'react'
```

Comment:
Could you change the imports to:

```ts
import { type Component, type ReactNode, createRef } from 'react'
```

Im going to end up adding the type keyword to all the type imports to make it more readable.